Quarkus CLI 3.25 unable to find mvn command (when no ./mvnw)

### Describe the bug

I'm using Quarkus CLI 3.25.0 to update my project from 3.15 to 3.20. I have been doing it version by version by incrementing --stream parameter from 3.16 to 3.17 to 3.18 and so on. After trying quarkus update --stream=3.19, I get an error unable to find the maven executable.
It is in my path, maven home is also set, `mvn -v` returns 3.9.6 and `where mvn` returns correct location.

### Expected behavior

`quarkus update --stream=3.19' would update the project to 3.19 as it did for versions before.

### Actual behavior

`quarkus update --stream=3.19` returns this error:
```
[ERROR]  Unable to find the maven executable, is it in your path?
[ERROR]  Unable to run Quarkus project update : Unable to find mvn command
```

`quarkus update --stream=3.19 -e -- verbose` returns this error:
```
[ERROR]  Unable to find the maven executable, is it in your path?
java.lang.RuntimeException: Unable to find mvn command
        at io.quarkus.devtools.exec.Executable.findExecutable(Executable.java:40)
        at io.quarkus.cli.build.ExecuteUtil.findExecutable(ExecuteUtil.java:30)
        at io.quarkus.cli.build.MavenRunner.getExecutable(MavenRunner.java:68)
        at io.quarkus.cli.build.BuildSystemRunner.prependExecutable(BuildSystemRunner.java:64)
        at io.quarkus.cli.build.MavenRunner.updateProject(MavenRunner.java:191)
        at io.quarkus.cli.Update.call(Update.java:25)
        at io.quarkus.cli.Update.call(Update.java:11)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
        at picocli.CommandLine.execute(CommandLine.java:2174)
        at io.quarkus.cli.QuarkusCli.run(QuarkusCli.java:120)
        at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:141)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:80)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:51)
        at io.quarkus.cli.Main.main(Main.java:9)

[ERROR]  Unable to run Quarkus project update : Unable to find mvn command
```

### How to Reproduce?

I cannot show you the project since its from work, but if I need to provide anything additionally let me know.

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

21.0.1

### Quarkus version or git rev

3.18.4

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.6

### Additional information

It seemed to work yesterday (29.7. 2025) fine, but this morning I get this error. When trying the command on coworkers laptop with CLI 3.25.0 installed, he gets the same error.

Thanks for the help

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -46,7 +46,7 @@
         <microprofile-jwt.version>2.1</microprofile-jwt.version>
         <microprofile-lra.version>2.0.1</microprofile-lra.version>
         <microprofile-openapi.version>4.0.2</microprofile-openapi.version>
-        <smallrye-common.version>2.13.7</smallrye-common.version>
+        <smallrye-common.version>2.13.8</smallrye-common.version>
         <smallrye-config.version>3.13.4</smallrye-config.version>
         <smallrye-health.version>4.2.0</smallrye-health.version>
         <smallrye-metrics.version>4.0.0</smallrye-metrics.version>

--- a/independent-projects/arc/pom.xml
+++ b/independent-projects/arc/pom.xml
@@ -49,7 +49,7 @@
         <version.jboss-logging>3.6.1.Final</version.jboss-logging>
         <version.mutiny>2.9.4</version.mutiny>
         <version.bridger>1.6.Final</version.bridger>
-        <version.smallrye-common>2.13.7</version.smallrye-common>
+        <version.smallrye-common>2.13.8</version.smallrye-common>
         <!-- test versions -->
         <version.assertj>3.27.3</version.assertj>
         <version.junit5>5.13.4</version.junit5>

--- a/independent-projects/bootstrap/pom.xml
+++ b/independent-projects/bootstrap/pom.xml
@@ -69,7 +69,7 @@
         <plexus-interpolation.version>1.26</plexus-interpolation.version>
         <plexus-sec-dispatcher.version>2.0</plexus-sec-dispatcher.version>
         <plexus-utils.version>3.5.1</plexus-utils.version>
-        <smallrye-common.version>2.13.7</smallrye-common.version>
+        <smallrye-common.version>2.13.8</smallrye-common.version>
         <smallrye-beanbag.version>1.5.3</smallrye-beanbag.version>
         <gradle-tooling.version>8.14</gradle-tooling.version>
         <quarkus-fs-util.version>1.1.0</quarkus-fs-util.version>

--- a/independent-projects/junit5-virtual-threads/pom.xml
+++ b/independent-projects/junit5-virtual-threads/pom.xml
@@ -42,7 +42,7 @@
         <enforcer.plugin.version>3.2.1</enforcer.plugin.version>
         <surefire.plugin.version>3.5.2</surefire.plugin.version>
         <jandex.version>3.4.0</jandex.version>
-        <smallrye-common.version>2.13.7</smallrye-common.version>
+        <smallrye-common.version>2.13.8</smallrye-common.version>
 
         <junit5.version>5.13.4</junit5.version>
         <assertj.version>3.27.3</assertj.version>

--- a/independent-projects/qute/pom.xml
+++ b/independent-projects/qute/pom.xml
@@ -43,7 +43,7 @@
         <version.jandex>3.4.0</version.jandex>
         <version.gizmo>1.9.0</version.gizmo>
         <version.jboss-logging>3.6.1.Final</version.jboss-logging>
-        <version.smallrye-common>2.13.7</version.smallrye-common>
+        <version.smallrye-common>2.13.8</version.smallrye-common>
         <version.smallrye-mutiny>2.9.4</version.smallrye-mutiny>
     </properties>
 

--- a/independent-projects/resteasy-reactive/pom.xml
+++ b/independent-projects/resteasy-reactive/pom.xml
@@ -56,7 +56,7 @@
         <jakarta.persistence-api.version>3.1.0</jakarta.persistence-api.version>
 
         <mutiny.version>2.9.4</mutiny.version>
-        <smallrye-common.version>2.13.7</smallrye-common.version>
+        <smallrye-common.version>2.13.8</smallrye-common.version>
         <vertx.version>4.5.16</vertx.version>
         <rest-assured.version>5.5.5</rest-assured.version>
         <commons-logging-jboss-logging.version>1.0.0.Final</commons-logging-jboss-logging.version>

